modules/era/lua/era_pet_reward.lua

```diff
@@ -0,0 +1,141 @@
+-----------------------------------
+-- Era pet reward ability
+-----------------------------------
+require('modules/module_utils')
+-----------------------------------
+local m = Module:new('era_pet_reward')
```

Comment:
This will be used for the entire beastmaster folder since it was converted to job_utils. Lets call this `era_beastmaster` maybe?

Reply:
https://github.com/LandSandBoat/server/pull/5532

yea - the calcs were being redone by Tib I believe. He was waiting for jimmy to get some data points... with that said, can close this one and pull LSB's - then module the parts we need to to make era appropriate.

Reply:
Sounds good!!